osbs/build/build_request.py

```diff
@@ -559,6 +563,19 @@ def adjust_for_custom_base_image(self):
             logger.info(msg, which)
             self.dj.remove_plugin(when, which)
 
+    def adjust_for_repo_info(self):
+        if not self._repo_info:
+            logger.warning('repo info not set')
+            return
+
+        if not self._repo_info.configuration.is_autorebuild_enabled():
+            logger.info('autorebuild is disabled in repo configuration, removing triggers')
+            self.template['spec'].pop('triggers', None)
+
+        elif 'release' in self._repo_info.dockerfile_parser.labels:
+            raise RuntimeError('when autorebuild is enabled in repo configuration, '
```

Comment:
It would do that just once, without writing the changes back to Dockerfile. As a result, next autorebuild of this would cause exception in `koji_promote`/`koji_tag_build` - as the build with the bumped release already exists.